cmd/src/snapshot_databases.go

```diff
@@ -49,7 +51,7 @@ TARGETS FILES
 
 			var builder string
 			if len(args) > 0 {
-				builder = args[0]
+				builder = args[len(args)-1]
 			}
```

Comment:
this was my mistake, I think we want to use `flagSet.Args()` to get the non-flag arguments: https://sourcegraph.com/github.com/golang/go@1fffeddfe9c977510d855277da57e0564700d6c3/-/blob/src/flag/flag.go?L626-627